config.toml

```diff
@@ -153,6 +153,45 @@ weight = 2
 home = "Accueil"
 copyright = "Copyright &copy; 2021 Rotational Labs, LLC, All Rights Reserved"
 
+## Chinese Navigation
```

Comment:
Would you mind reverting the changes made to this file so that we can adopt the go-18n for Hugo approach in a subsequent PR?